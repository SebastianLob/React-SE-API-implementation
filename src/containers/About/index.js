import React from 'react';
import Container from 'react-bootstrap/Container';
import Row from 'react-bootstrap/Row';
import Col from 'react-bootstrap/Col';

export default function About() {
  return (
    <Container fluid>
      <Row>
        <Col md={3} />
        <Col md={6}>
          This site was created for implementing the Google and Bing Search
          engines API's. It was created by{' '}
          <a href='https://www.linkedin.com/in/agustinknaupp/'>
            Agustin Knaupp
          </a>
          .
        </Col>
        <Col md={3} />
      </Row>
    </Container>
  );
}
