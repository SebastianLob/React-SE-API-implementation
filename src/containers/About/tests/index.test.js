import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import About from '../index';

describe('Basic Pagination', () => {
  const renderer = new ShallowRenderer();
  it('should render and match the snapshot', () => {
    renderer.render(<About />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
