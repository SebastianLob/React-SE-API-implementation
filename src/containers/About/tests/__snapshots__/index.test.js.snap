// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic Pagination should render and match the snapshot 1`] = `
<Container
  fluid={true}
>
  <Row
    noGutters={false}
  >
    <Col
      md={3}
    />
    <Col
      md={6}
    >
      This site was created for implementing the Google and Bing Search engines API's. It was created by
       
      <a
        href="https://www.linkedin.com/in/agustinknaupp/"
      >
        Agustin Knaupp
      </a>
      .
    </Col>
    <Col
      md={3}
    />
  </Row>
</Container>
`;
